backend/contributions/migrations/0069_alter_contributiondiscordxpstate_options_and_more.py

```
@@ -0,0 +1,60 @@
+    )
+    ContributionDiscordXPState.objects.bulk_create([
+        ContributionDiscordXPState(
+            social_task_completion=completion,
+            status='pending',
+            awarded_amount=0,
+        )
+        for completion in completions.iterator()
+    ], batch_size=500)
+
+
+def remove_social_task_xp_states(apps, schema_editor):
```

Ruff (0.15.15)

[warning] Unused function argument: `schema_editor`

(ARG001)